src/frontend/components/PublicPausedLiveVideo/index.spec.tsx

```diff
@@ -3,10 +3,29 @@ import { DateTime } from 'luxon';
 import React from 'react';
```

Comment:
in commit message videojs stop **fetching 
Doing** this fetche**s** the
manifest and resume**s** the live where it was paused.